current/fyne/builtins_fyne.go

```diff
@@ -195,22 +195,113 @@ var Builtins_fyne = map[string]*env.Builtin{
 			return *env.NewNative(ps.Idx, win, "fyne-widget")
 		},
 	},
+
+	"spacer": {
+		Argsn: 0,
+		Doc:   "Creates Fyne layout spacer",
+		Fn: func(ps *env.ProgramState, arg0 env.Object, arg1 env.Object, arg2 env.Object, arg3 env.Object, arg4 env.Object) env.Object {
+			win := layout.NewSpacer()
+			return *env.NewNative(ps.Idx, win, "fyne-layout-spacer")
+
+		},
+	},
+
 	"container": {
+		Argsn: 1,
+		Doc:   "Creates Fyne container",
+		Fn: func(ps *env.ProgramState, arg0 env.Object, arg1 env.Object, arg2 env.Object, arg3 env.Object, arg4 env.Object) env.Object {
+			switch bloc := arg0.(type) {
+			case env.Block:
+				items := []fyne.CanvasObject{}
+				for _, it := range bloc.Series.S {
+					switch nat := it.(type) {
+					case env.Native:
+						items = append(items, nat.Value.(fyne.CanvasObject))
+					}
+				}
+				win := container.NewWithoutLayout(items...)
+				return *env.NewNative(ps.Idx, win, "fyne-container")
+			default:
+				return evaldo.MakeArgError(ps, 2, []env.Type{env.WordType}, "fyne-container")
+			}
+		},
+	},
+
+	"container-vbox": {
+		Argsn: 1,
+		Doc:   "Creates Fyne vbox container",
+		Fn: func(ps *env.ProgramState, arg0 env.Object, arg1 env.Object, arg2 env.Object, arg3 env.Object, arg4 env.Object) env.Object {
+
+			switch bloc := arg0.(type) {
+			case env.Block:
+				items := []fyne.CanvasObject{}
+				for _, it := range bloc.Series.S {
+					switch nat := it.(type) {
+					case env.Native:
+						items = append(items, nat.Value.(fyne.CanvasObject))
+					}
+				}
+				win := container.NewVBox(items...)
+				return *env.NewNative(ps.Idx, win, "fyne-container-vbox")
+			default:
+				return evaldo.MakeArgError(ps, 2, []env.Type{env.WordType}, "fyne-container-vbox")
+			}
+		},
+	},
+
+	"container-hbox": {
+		Argsn: 1,
+		Doc:   "Creates Fyne hbox container",
+		Fn: func(ps *env.ProgramState, arg0 env.Object, arg1 env.Object, arg2 env.Object, arg3 env.Object, arg4 env.Object) env.Object {
+			switch bloc := arg0.(type) {
+			case env.Block:
+				items := []fyne.CanvasObject{}
+				for _, it := range bloc.Series.S {
+					switch nat := it.(type) {
+					case env.Native:
+						items = append(items, nat.Value.(fyne.CanvasObject))
+					}
+				}
+				win := container.NewHBox(items...)
+				return *env.NewNative(ps.Idx, win, "fyne-container-hbox")
+			default:
+				return evaldo.MakeArgError(ps, 2, []env.Type{env.WordType}, "fyne-container-hbox")
+			}
+		},
+	},
+
+	"container-grid-rows": {
```

Comment:
`container-grid-rows` seems long .. do you have any better (shorter) idea how to name them, I don't :(

Reply:
We can  start with full names and we will see with use. Or I see option that we remove all "container-" and just leave the specific container type: container-hbox -> hbox , container-vbox -> vbox , container-grid -> grid, so then this would be grid-rows / grid-cols which I think is quite nice.